docs/source/quark_script.rst

```diff
@@ -207,6 +207,13 @@ methodInstance.getArguments(none)
 - **Description**: Get arguments from method.
 - **params**: none
 - **return**: python list containing arguments.
+  
+methodInstance.findSuperclassHierarchy(none)
+==============================
+
+- **Description**: Find all superclass hierarchy of this method object.
+- **params**: none
+- **return**: Python list contains all superclas’s name of the this method.  
```

Comment:
The apostrophe in "superclas's" should be placed at the end of the word rather than the second-last letter.